src/bitcoin/mod.rs

```diff
@@ -502,7 +502,13 @@ impl BitcoinInterface for electrum::Electrum {
                     expired_spending.push(*op);
                 }
                 if let Some(block) = w_c.spend_block {
-                    spent.push((*op, *spend_txid, block.height, block.time));
+                    spent.push((
+                        *op,
+                        w_c.spend_txid
+                            .expect("must have spend_txid if spend_block present"),
```

Comment:
Can we wrap it in an `if let Some(coin_spend_txid) = ` and `log::error` in the `else` branch instead, just in case?

Reply:
Sure.

Reply:
Done.